### PR TITLE
feat: apply page access permissions

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### CODE REVIEW

File: apps/web/src/services/iam/components/RoleUpdateForm.vue [hunk 1]
```diff
@@ -37,7 +37,11 @@ const state = reactive({
     roleTypeData: ROLE_TYPE.DOMAIN_ADMIN as RoleType,
     pageAccessFormData: ['*'] as string[],
     permissionsData: undefined as string[]|undefined,
-    isAllValid: computed<boolean>(() => state.isBaseInformationValid),
+    isPageAccessValid: false,
+    isAllValid: computed<boolean>(() => {
+        const isPageAccessCheckRequired = state.roleTypeData !== 'DOMAIN_ADMIN';
```
Comment (sulmoJ) [inline apps/web/src/services/iam/components/RoleUpdateForm.vue:43]: The constant ROLE_TYPE exists. Please change it to a constant!

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Review by sulmoJ — APPROVED** [review]
I left nitpick comment!
LGTM~

**Review by piggggggggy — APPROVED** [review]
Goood~

**Pull request merged**